src/Sulu/Bundle/MediaBundle/Domain/Event/MediaVersionCreatedEvent.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Domain\Event;
+
+use Sulu\Bundle\EventLogBundle\Domain\Event\DomainEvent;
+use Sulu\Bundle\MediaBundle\Admin\MediaAdmin;
+use Sulu\Bundle\MediaBundle\Entity\Collection;
+use Sulu\Bundle\MediaBundle\Entity\FileVersionMeta;
+use Sulu\Bundle\MediaBundle\Entity\MediaInterface;
+
+class MediaVersionCreatedEvent extends DomainEvent
```

Comment:
i know i suggested this name initially, but i think i would call this `MediaVersionAddedEvent` 🙈  what do you think?

Reply:
`added` would imply that it is added to an existing entity and would be consistent to the `translation_added` type. in hindsight, i think i would also use `keyword_added` instead of `keyword_created` in the category bundle now

Reply:
for completion, think i will create a pr after this to rename the `keyword_created` event 😅